### PR TITLE
dev-uxmt: Prepare integration tests for Github actions.

### PR DESCRIPTION
**Pull request by jpkrajewski**
# Pull Request summary:
- Prepare integration tests for github actions
- Make tests work for vmanage ver. 20.12
- Add storing and encrypting artifacts and log
- Add constrain that integration tests will be run only in allowed window (10 PM - 7 AM PDT) [sharing that resource based on time table]

To decrypt use:
```
gpg --decrypt --passphrase="YOUR_PASSPHRASE" archive.tar.gz.gpg > decrypted_archive.tar.gz
```

# Checklist:
- [ ] All testcases suited for 20.12
- [ ] Storing artifactsand encrypting artifacts

### CODE REVIEW

File: .github/workflows/integrationtests.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      - if: ${{ env.ACT }}
+        name: Hack container for local development
+        run: |
+            curl -fsSL https://deb.nodesource.com/setup_12.x | sudo -E bash -
+            sudo apt-get install -y nodejs
+      - uses: actions/checkout@v4
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: 3.10
```
Comment (sbasan) [inline .github/workflows/integrationtests.yml:30]: maybe 3.11 (as in cmt?)

Comment (jpkrajewski) [reply .github/workflows/integrationtests.yml]: Ok

File: .github/workflows/integrationtests.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
+on:
+  pull_request:
+    branches:
+      - dev-uxmt
+
```
Comment (sbasan) [inline .github/workflows/integrationtests.yml:5]: maybe allow only one job concurrently as it is being run on shared resource
https://docs.github.com/en/actions/using-jobs/using-concurrency

Comment (jpkrajewski) [reply .github/workflows/integrationtests.yml:5]: Done

**Review by sbasan — APPROVED** [review]
lgtm

**Pull request merged**